src/components/BeverageDetailsLoader/LoadedBeverageDetails.js

```diff
@@ -34,22 +35,24 @@ const LoadedBeverageDetails = ({
   return (
     <View>
       <SectionHeader title={name} />
-      <CacheImage
-        style={{
-          height: screenWidth * 0.75,
-          width: screenWidth,
-        }}
-        source={{
-          uri: `${CONFIG.CDN}beverages/${id.toString()}-icon.jpg?w=${
-            beverageImageWidth
-          }&h=${beverageImageHeight}&mode=crop`,
-        }}
-      />
-      <Text style={styles.descriptionText}>{description}</Text>
-      <OverviewItem title="Type" value={beverageType} />
-      {style && <OverviewItem title="Style" value={style.name} />}
-      {glass && <OverviewItem title="Glass" value={glass.name} />}
-      <OverviewItem title="Organic?" value={isOrganic ? 'Yes' : 'No'} />
+      <SectionContent>
+        <CacheImage
+          style={{
+            height: screenWidth * 0.75,
```

Comment:
What is this doing here? I just want to make sure that the aspect ratio doesn't get distorted.